packages/patrol_cli/lib/src/dart_defines_reader.dart

```diff
@@ -25,6 +27,19 @@ class DartDefinesReader {
     return _parse(lines);
   }
 
+  Map<String, dynamic> fromConfigFile({required String path}) {
+    final filePath = _fs.path.join(_projectRoot.path, path);
+    final file = _fs.file(filePath);
+
+    if (!file.existsSync()) {
+      return {};
+    }
+
+    final jsonString = file.readAsStringSync();
+    final json = jsonDecode(jsonString) as Map<String, dynamic>;
+    return json;
+  }
+
```

Comment:
If the decision was made to completely parse the files into separate `define`'s, then I would like it to work the same way as in `flutter_tools`:
- fail if file is missing
- do not add `project Root.path` at beginning of file path
- support more than just `JSON` (maybe someone is using this)

How this is implemented in `flutter_tools`:
https://github.com/flutter/flutter/blob/64dd1ca9bc32fbc767a3f63e4c9c5ae2fc355380/packages/flutter_tools/lib/src/runner/flutter_command.dart#L1452-L1494

Reply:
Thanks. Good notice! I reused flutter_tools code for parsing. You can check it out now